### ISSUE TITLE
Missing aria-label on close (X) button on accessibility plugin

### ISSUE THREAD

**Issue by ghahemcl** [ponlawat-w/moodle-local_accessibility#49]
The close (X) button used in the accessibility plugin does not include an aria-label or appropriate accessibility attributes. This makes it difficult for screen reader users to understand the purpose of the button, which is a key accessibility concern.

See code: <a href="javascript:void(0);" id="local-accessibility-closebtn" class="float-right text-light text-decoration-none">
<i class="fa fa-times"></i>
</a>
